examples/material-designer-workflow/assets/project.wbjn

```diff
@@ -0,0 +1,17 @@
+import os
+import json
+import string
+import os.path
+
+work_dir = GetServerWorkingDirectory()
+arg_ProjectArchive = os.path.join(work_dir, "MatDesigner.wbpz")
+
+# Description="Insert Steady State Thermal linked with Static Structural system and define blade and fluid materials"
+Unarchive(
+    ArchivePath=arg_ProjectArchive,
+    ProjectPath=GetAbsoluteUserPathName(work_dir + "wbpj\MatDesigner.wbpj"),
+    Overwrite=True)
+
+# output system name
+#result = system.Name
+#wb_script_result = json.dumps(result)
```

Comment:
These lines are commented out. Why not just remove them to keep the script tidy. @JimmyHe0527

Reply:
Thanks for pointing this out. I have removed this and cleaned up the comment.